airflow/configuration.py

```diff
@@ -430,10 +430,13 @@ def load_test_config(self):
         Note: this is not reversible.
         """
         # override any custom settings with defaults
+        log.info("Overriding settings with defaults from %s", DEFAULT_CONFIG_FILE_PATH)
```

Comment:
I think these should be debug -- it's a bit too noisy and not useful to see this all the time.

Reply:
Oh nm, this is load_test_config only :)